docs/specs/phasing-vector.md

```diff
@@ -0,0 +1,254 @@
+# Phasing (vector) pipeline specification
+
+* Version: 0.0.0
+* Authors: Alistair Miles, Jon Brenas
+* Status: DRAFT
+
+This document specifies a pipeline for phasing SNP genotypes for a
+cohort of multiple samples. The pipeline makes use of both read-backed
+phasing and statistical (population) phasing to improve phasing
+accuracy.
+
+
+## Inputs
+
+* A manifest specifying a set of samples to be phased.
+
+* For each sample, a BAM file containing analysis-ready sequence read
+  alignments, produced by the short read alignment pipeline.
+
+* For each sample, a VCF file or zipped zarr file containing unphased
```

Comment:
Do we have a spec for that pipeline yet?

Reply:
Yep, #20, added link.